images/node/Dockerfile

```diff
@@ -16,14 +16,12 @@ COPY system-container/manifest.json system-container/config.json.template system
 RUN INSTALL_PKGS=" \
       origin-hyperkube origin-node origin-sdn-ovs \
       socat ethtool device-mapper iptables nmap-ncat e2fsprogs \
-      xfsprogs device-mapper-persistent-data ceph-common \
+      xfsprogs device-mapper-persistent-data\
```

Comment:
Why did you remove this here?

Reply:
Because node is FROM openshift/origin-control-plane

Reply:
It has to remain here.  Node image won’t depend on control plane next release